### DISCUSSION TITLE
Cannot call the cli with "tc"

### DISCUSSION THREAD

**Discussion by JamesSalzer** [thunderclient/thunder-client-support#1007] [Q&A] [answered]
I'm not sure, maybe I'm doing something wrong, But it seems that the CLI does not work for me. Our
 environment runs via a WSL installation, when I type tc in the terminal I get response from "tc utility, iproute2-ss200127" a part of the Linux installation. I have attached a screenshot to illustrate it.

![tc_wsl](https://user-images.githubusercontent.com/115971524/217548431-de06ccf7-c85e-4d2a-8a78-b2e08dae3416.jpg)

**Comment by rangav** [comment]
Looks like you have another cli tool with tc prefix. 

Is it possible to uninstall that existing cli tool?

**Comment by rangav** [comment]
@JamesSalzer please install the latest version of cli  `v1.0.3`. 

Now you can use either `tc` or `tc2`.

let me know your feedback.

**Comment by JamesSalzer** [comment]
I have installed the CLI version 1.0.3, seems to be no more conflict with the Linux part.

But how it works I have probably not yet fully understood.
I show a screenshot, there you should see my problem.

![cli](https://user-images.githubusercontent.com/115971524/217762967-acd52e15-cf06-4688-8f36-876c9595a861.jpg)

**Reply by rangav** [reply]
Try using single or double quote around name 

`tc —col "Laravel_Routen”`

**Comment by JamesSalzer** [comment]
No, that's probably not the problem.

![cli2](https://user-images.githubusercontent.com/115971524/217766553-761101c9-e9cd-4107-8c6a-f218d6d9703f.jpg)

I am the rest of the week in the home office this week but on Monday I can try it on a colleague's PC, my computer is maybe a bit cluttered from installing and uninstalling a lot.

**Reply by rangav** [reply]
Where is your requests data is stored?

looks like its unable to find data

**Comment by JamesSalzer** [comment]
The data is together with the currently used branch on the company server and I access it via our Windows Subsystem for Linux (WSL).
The branch being processed is in Apache and the MySql DB on the server in wsl, the localhost address is redirected to wsl.
I hope this is correct as our admin is taking care of it.

**Reply by rangav** [reply]
Thanks for sharing. Is your TC extension is version 2.4.1 right?

**Reply by JamesSalzer** [reply]
Yes

**Reply by rangav** [reply]
Are you using any Git Sync settings?

https://github.com/rangav/thunder-client-support#git-sync

**Reply by rangav** [reply]
Also run `tc —debug` and let me know the output

**Comment by JamesSalzer** [comment]
yes, Thunder client data is in Git, here again a screenshot

![client3](https://user-images.githubusercontent.com/115971524/217782622-9742511b-8b2d-4f35-a209-daf5e07cc0d4.jpg)

**Reply by rangav** [reply]
ok I think I understood the problem.

The 2 settings `Save to Workspace` and `Workspace Relative Path` should be in `workspace` scope, so the settings are saved in .vscode folder in the project.

Then those settings can be accessed from cli also.

So can you please change the scope of all the TC settings you are using to workspace.

**Reply by rangav** [reply]
<img width="848" alt="Screenshot 2023-02-09 at 11 43 40" src="https://user-images.githubusercontent.com/8637550/217803782-0acdb33b-82c8-47de-8f00-ce3bc0c7f57c.png">

**Comment by JamesSalzer** [comment, marked as answer]
Great, now everything works.
I just had to move the Jason and .db files to the folder in the root directory, and now it works. Thank you.

**Reply by rangav** [reply]
Thanks for confirmation.

let me know any feedback to further improve CLI tool.
